### ISSUE TITLE
Updated Sign In Page Review from Login.gov Adoption Team

### ISSUE THREAD

**Issue by djmunoz1** [department-of-veterans-affairs/va.gov-team#62974]
## About your team

- Team name: Login.gov Adoption
-	OCTO-DE product owner: John Rahaghi
-	Product manager: Dawn Muñoz
-	Designer: Carolyn Williams
- Accessibility specialist: N/A
- Content writer: Carl Dickerson
-	Slack channel: #va-identity-product-team
 
### Describe what you need in a few sentences:
We want to implement an updated sign in page to prepare visitors for impending changes to credential options. These changes will intentionally push the modern credentials to the visual fore and provide written context to the Veteran and we need:
- to make sure the content we’re using in our tool is written in plain language and follows the style guide.

### Which are you doing?
- [ ] **Launching a new** page, tool, form, or app
- [ x] **Making changes** to an existing page, tool, form, or app

### Types of Support
What areas does your team need support in? Check all that apply.
#### Content
- [x ] Collaboration on content written by your team’s dedicated content writer 
- [ ] Collaboration on content if your team does not have a dedicated content writer
#### Accessibility
- [x ] Accessibility review of wireframes and/or prototypes
- [ ] Accessibility issue (general request)
- [ ] Accessibility PDF review request
- [ ] Accessibility user research support ([submit a separate, additional request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name))
- [ ] DST component or pattern 
- [ ] Something else 
#### IA
- [ ] User Flows
- [ ] Page placement
- [ ] Object-Oriented UX (OOUX) support
- [ ] URLs
- [ ] Breadcrumbs and Left Nav placement
- [ ] Redirects
#### Multilingual content
- [ ] Translating content
 
## Timeframe
_We guide and work alongside teams across nearly every OCTO digital product. We also work with partners across VA to lead content migration and manage all unauthenticated content on VA.gov. And we're always working to expand the VA.gov content style guide and our IA and accessibility documentation. We need to prioritize all intake requests based on our overall workload and VA and OCTO priorities._

_Tell us about your product's timeline. And we'll work with you to meet timeline needs as best we can._
We need to make these changes promptly in support of CIO and CTO level goals for the sunset of legacy credentials. Requesting review within 3 business days of receipt of this request.

### Have you worked with CAIA before? 
  - [x] Yes
  - [ ] No

### What's your team’s next step and its timing?

- **Our next step is** to complete the Collab cycle requirements and preview this updated page to a limited number of visitors (up to 50k initially) before widening the funnel incrementally.
- **Provide date if available**: Target launch 8/14/23

#### Timing for your next step: 
  - [ ] Next week
  - [X] This month
  - [ ] Next month
  - [ ] This quarter
  - [ ] Next quarter
  - [ ] 6 months
  - [ ] 1 year

### Is this timing related to a specific event or Congressionally mandated deadline?
  - [X ] Yes; EO Identity Fraud
  - [] No

### If you're conducting research, when is that starting? 
Following discussions with our PO, we will not conduct individual research however we will be reviewing feedback following each phase of the rollout to inform iterative changes and would like to also review Medallia survey feedback in conjunction with this update.

**Provide date if available:** xx/xx/xxxx
  - [ ] Next week
  - [ ] This month
  - [ ] Next month
  - [ ] This quarter
  - [ ] Next quarter
  - [ ] 6 months
  - [ ] 1 year

### Will you release this new product incrementally (for example, release to 25% of users to start)? 
  - [x] Yes
  - [ ] No
 
**Note**: _If you checked yes, we’ll reach out to discuss details about the content in the react widget (we use these widgets to display entry points for new products to a certain percentage of users who visit our static pages)._

  
### When do you expect to launch your product to 100% of users? September (date TBD depending on outcomes of phased releases)
  
*Please provide an estimated date so our team can create other relevant tickets.*
- **We expect to launch our product to 100% of users on:**  Est mid Sept
  
### Will this work be going through the Collaboration Cycle?
- [x] Yes 
- [ ] No

### If no ...
- Skip to the _supporting artifacts_ section, below.
 
#### Please provide the link to your Collaboration Cycle ticket: 
- **Ticket (number)**: [#62910](https://github.com/department-of-veterans-affairs/va.gov-team/issues/62910)
- **Collab. Cycle Ticket Opened (date)**: 7/31/23
 
#### Please check all of the phases in the Collaboration Cycle you have completed: 
- [ ] Design Intent
- [ ] Midpoint Review
- [ ] Staging
 
#### For planning purposes, please check all of the phases in the Collaboration Cycle you feel you might need CAIA support and guidance: 

This is a change to a single static page related to VA Profile, as such we anticipate minimal cycles and efficient review as the platform team deems necessary.

##### Design Intent
- [ ] Accessibility
- [ ] Content
- [ ] IA
- [ ] Translation
- [ ] Already completed, no assistance needed
- [ ] Unknown at this time
##### Midpoint
- [ ] Accessibility
- [ ] Content
- [ ] IA
- [ ] Translation
- [ ] Already completed, no assistance needed
- [ ] Unknown at this time
##### Staging
- [ ] Accessibility
- [ ] Content
- [ ] IA
- [ ] Translation
- [ ] Already completed, no assistance needed
- [ ] Unknown at this time
  
## Supporting artifacts
*Please provide links to supporting artifacts as available.*
- **Product outline**: https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/login.gov-adoption/products/login.gov
- **User flows**:
- **Prototype or mockups**: pls request access when needed https://www.figma.com/file/8vobVoSl1UqLnXsni3rcvO/Login.gov-Adoption%3A-Design-Flows?type=design&node-id=2601-33&mode=design&t=BkZAvIjjrNBQDAUo-0
- **If this is in Sketch, which version or specific page should we focus on?**
- **Content source of truth**:
- **Research plan:**
- **Any epics/issues that may be helpful**:
- **Link to testing accounts** (please do not paste them in this ticket): 

### Have you already worked with someone from OCTO leadership?
- [ ] Danielle
- [ ] Beth
- [ ] Mikki
- [ ] Martha

### How do you plan to work with us on this request?
- [ ] I’ll take this to Beth or Danielle’s office hours.
- [ ] I’d like to schedule time to talk about this.
- [x] Let’s work async and meet as needed.

 
### Accessibility Help in Slack
The [#accessibility-help channel](https://dsva.slack.com/archives/C8E985R32) in the DSVA Slack, is also available to you as a resource. Everyone is welcome to ask questions or get help from our a11y (accessibility) specialists. Tag `@Terry Nichols` to get a11y help asap.

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#sitewide-content-ia](https://dsva.slack.com/channels/sitewide-content-ia) Slack channel and tag `@Terry Nichols`.**

- If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
- If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request)
 
 - If you are creating an [experimental design,](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) please [also contact the DST. ](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)

- If you are requesting [a11y support for user research,](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name) please complete this form, next.

**Comment by kristinoletmuskat** [comment]
Untagging IA - confirmed no IA work needed!

**Comment by laurwill** [comment]
Hi @djmunoz1 ,
Here's our suggested content for this info alert, based on live content from the Creating an account page:

> We encourage you to sign in using a **Login.gov** or **ID.me** account. These accounts meet modern security standards. If you don’t have either of these accounts, you can create a free **Login.gov** or **ID.me** account now.

Here's some notes about this approach:
- This alert isn't targeted messaging, so it needs to address people who have and who don't have these account types already. We used the "if" statement at the end to do this.
- Based on feedback in testing, we know that messages that highlight a need to make specific changes by a specific time will be more motivating. But we don't recommend talking about an upcoming change in this context, until we're ready to more clearly describe what will happen. 
- In this case, it's difficult to say what the upcoming change is without directly saying that DS Logon and MHV accounts will no longer be sign-in options (which we haven't heard that we're ready to do). So we've pulled back to a very general statement about the account types we recommend.
- We haven't included a link to the Creating an account page in the info alert, because we don't want to pop people out of the sign-in flow. 

And here's a few other considerations:
- Danielle let me know that a My HealtheVet coordinator raised a concern (which we also heard in testing) about people thinking Login.gov and ID.me are tools or websites, rather than account types. We might want to consider making the button text more explicit about each option being an account.
- I'm concerned that people who currently sign in with DS Logon or MHV may be confused by the change from buttons to text links, and think these options are already gone. Is that something you'll be assessing as part of the incremental release? 
- I wondered if a full alert style with a header would be more noticeable here (rather than background only) — but we'll defer to design guidance in collab cycle.

Thanks for your collaboration on this work! Let me know if you have any questions about this approach, and happy to talk more next week.

cc @DanielleThierryUSDSVA for awareness

**Comment by DanielleThierryUSDSVA** [comment]
Hi @laurwill @djmunoz1 and team. Please do not share this content as final with anyone. I am re-routing this through the newly formed unified MHV on VA.gov + Sign in and identity communications working group. Thank you!
